src/rateEngine/LoadProfile.ts

```diff
@@ -97,6 +99,20 @@ class LoadProfile {
     });
   }
 
+  maxPerDayByMonth(): Array<number> {
```

Comment:
I'm open to a better name for this method if one stands out @bknoles @euggra

Reply:
That's a great name.

Reply:
I see why this gives you pause... it's missing the "sum" concept. something like `maxPerDaySummedByMonth` is more accurate. That's a pretty awkward name, which makes me think there's probably another class we could add.

Something that would yield an API like:

```
loadProfile.max(by: 'day').sum(by: 'month');
// or
loadProfile.maxByDay().sumByMonth();
```

That's out of the scope of this PR, but i would be in favor of using the awkward name until we can take a longer look: `maxPerDaySummedByMonth`.